utils/getFeed.js

```diff
@@ -20,10 +22,9 @@ import { parseFeedItem as parsePostFeedItem } from "./getPostFeed";
  * @returns {string|undefined} The string result of the xpath query.
  */
 export function getXPathText( xpath, document, context = null, nsResolver = null ) {
-	const tagname = xpath.replace( "child::", "" );
-	const elementExists = document.getElementsByTagName( tagname );
+	const elementCount = document.evaluate( "count(" + xpath + ")", ( context || document ), nsResolver, XPathResult.ANY_TYPE, null ).numberValue;
```

Comment:
Please add a comment about what you're doing here/what you're doing here. It's hard to understand what's happening here